app/Http/Resources/ReportedUserReportResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Models\User;
+
+class ReportedUserReportResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'comment' => $this->comment,
+            'reported_date' => $this->created_at->toDateTimeString(),
+            'reported_by_id' => $this->reported_by_user_id,
+            'reported_by_name' => User::find($this->reported_by_user_id)->username,
```

Comment:
Change this to `User::find($this->reported_by_user_id)->value('username')` so you only collect one item, rather than the entire user. I know I haven't done this elsewhere myself, but it's a usertstory that I'm gonna work on soon.

Reply:
changed to `$this->getReportedUser()->value('username')`
and added
`public function getReportedUser() {
    return User::find($this->reported_user_id);
}`
to the `ReportedUser` model